lib/engines/content_block_manager/app/components/content_block_manager/shared/embedded_objects/summary_card/nested_item_component.rb

```diff
@@ -21,10 +25,14 @@ def title
   end
 
   def rows
-    nested_items.map do |key, value|
+    nested_items.map do |field, value|
```

Comment:
Wonder if we should rename this to `field_name`, so we're clearer this will be a string, rather than a field object.